main/src/init/file_init.hpp

```diff
@@ -106,7 +106,6 @@ class FileInit : public ISimInitializer<Dataset>
         initField(h5_file, rank, d.vy, "vy", 0.0);
         initField(h5_file, rank, d.vz, "vz", 0.0);
 
-        initField(h5_file, rank, d.dt_m1, "dt_m1", d.minDt);
         initField(h5_file, rank, d.du_m1, "du_m1", 0.0);
```

Comment:
Why do we need to store the _m1 fields in the HDF5? Usually, it is enough to initialize them with the same value as the current states and use a small timestep.

Reply:
You don't need to. You can choose not to store them and reset the timestep to something small.
If you want to restore your system to the exact same state, then they are needed.